index.html

```diff
@@ -38,6 +38,27 @@ <h1>Skinclusive<br>inclusive of your skin.</h1>
 				<p>This box will have buttons and text</p>
 			</div>
 
+			<!-- This is the individual text box-->

```

Comment:
I used layout 1's main design but I also decided to include the design of the top articles as seen in layout 2 as I thought it seemed quite fitting for the home page.